docs/orchestration/rbac/overview.md

```diff
@@ -127,3 +127,26 @@ query {
 ```
 
 The `memberships.id` value for a given user should be provided as the `membership_id` parameter when calling `set_membership_role`.
+
+### Audit Log (Enterprise Only)
+
+Separately from task run or flow run logs, Prefect has an Enterprise only feature to track “audit logs” for events within the system per tenant. 
+
+Audit logs for each tenant have a `timestamp`, `message` describing the event, an `object_id` associated with an `object_table` and an option `info`.
```

Comment:
```suggestion
Audit logs for each tenant have a `timestamp`, a `message` describing the event, an `object_id` associated with an `object_table`, and an option `info`.
```

Reply:
is it "optional 'info'"?

Reply:
I would say something like "an optional `info` field, which contains information about who performed an action (e.g. the user who turned a flow's schedule on)."